src/main/cli/settings.c

```diff
@@ -1131,7 +1131,11 @@ const clivalue_t valueTable[] = {
 
     { "angle_level_strength",       VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].P) },
     { "horizon_level_strength",     VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].I) },
-    { "horizon_transition",         VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].D) },
+    { "angle_level_derivative_gain",VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_LEVEL].D) },
+    { "horizon_transition",         VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, horizonTransition) },
```

Comment:
Now's also a good time to put underscores between variable names, to match the text in the cli.  In this case, to globally change the variable name `horizonTransition` to `horizon_transition`.  This simplifies searching for code that uses this variable or CLI entry and is why the majority of new settings are done like that.

Reply:
I'm not sure it's a good idea to rename existing variables while adding new features. I would propose that is done wholesale in a separate PR that only deals with renaming variables.